q2_vsearch/_join_pairs.py

```diff
@@ -139,8 +139,6 @@ def _join_pairs_w_command_output(
             cmd += ['--fastq_maxmergelen', str(maxmergelen)]
         if maxee is not None:
             cmd += ['--fastq_maxee', str(maxee)]
-        if threads is not None:
-            cmd += ['--threads', str(threads)]
```

Comment:
Sorry you still need this `cmd += ...` bit, it just doesn't need to be in a branch.